pkgs/applications/science/math/gmsh/default.nix

```diff
@@ -1,27 +1,50 @@
 { lib, stdenv, fetchurl, cmake, blas, lapack, gfortran, gmm, fltk, libjpeg
-, zlib, libGL, libGLU, xorg, opencascade-occt }:
+, zlib, libGL, libGLU, xorg, opencascade-occt
+, python, enablePython ? false }:
```

Comment:
I think you need `, python ? null`: `python` is `python-2.7` outside python modules. As a result `nixpkgs-review` fails as:

```
$ nixpkgs-review pr --no-shell --post-result --build-args "--max-jobs 1" 208396
...
error: Function called without required argument "python" at /home/slyfox/.cache/nixpkgs-review/pr-208396/nixpkgs/pkgs/applications/science/math/gmsh/default.nix:3, did you mean "jython", "python2" or "python3"?
(use '--show-trace' to show detailed location information)
nix --experimental-features nix-command --system x86_64-linux eval --json --impure --no-allow-import-from-derivation --expr (import /nix/store/wlrm50hify66j3jcbqifw1hgywim7i8q-nixpkgs-review-2.8.0/lib/python3.10/site-packages/nixpkgs_review/nix/evalAttrs.nix { allowAliases = false; attr-json = /tmp/tmp8g42oyj_; }) failed to run, /tmp/tmp8g42oyj_ was stored inspection
https://github.com/NixOS/nixpkgs/pull/208396 failed to build
$ git worktree prune
```

Otherwise the change looks good.